README.md

```diff
@@ -1,3 +1,131 @@
 # testcafe-browser-natives
 
 [![Build Status](https://travis-ci.org/superroma/testcafe-browser-natives.svg)](https://travis-ci.org/superroma/testcafe-browser-natives)
+
+Testcafe Browser Natives is a TestCafe library for performing platform-dependent actions on browser windows.
+#Build Process
+To build the binaries from source files, execute the gulp task corresponding to your operating system:
+```
+'copy-win-executables'
+'build-mac-executables'
+'copy-mac-script'
+```
```

Comment:
We write about binaries here. The `build-win` builds the `lib` dir also. If it does not matter we can write `build-win`